applications/sintering/include/pf-applications/sintering/mobility.h

```diff
@@ -595,7 +595,7 @@ namespace Sintering
                                  const Tensor<1, dim, VectorizedArrayType> &c_grad) const
     {
       // Surface anisotropic part
-      VectorizedArrayType fsurf = Msurf * (c * c) * (1. - c) * (1. - c);
+      VectorizedArrayType fsurf = Msurf * 4.0 * (c * c) * (1. - c) * (1. - c);
```

Comment:
How does this influence the number of iterations?

Reply:
This strictly speaking should not influence, but it can alter the behavior of the solver a bit (including the change of the linear/nonlinear iterations and/or time-step size) since now the surface diffusion is 4 times more intense than it was. This is for adequate comparison with the scalar case.

Reply:
Let me run one simulation before merging.